src/components/pages/donation_form/SubPages/DonationForm.vue

```diff
@@ -25,19 +25,29 @@
 			@set-address-type="setAddressType( $event )"
 		/>
 
-		<div class="content-card">
-			<FormSummary>
-				<template #summary-content v-if="paymentSummary">
-					<DonationSummary
-						:address="addressSummary"
-						:payment="paymentSummary"
-						:bank-data="bankDataSummary"
-						:countries="countries"
-						:salutations="salutations"
-					/>
-				</template>
-
-				<template #summary-buttons>
+		<ContentCard>
+			<template #heading v-if="paymentSummary">
+				<h2>{{ $t( 'donation_form_summary_title' ) }}</h2>
+			</template>
+			<template #content>
+				<DonationSummaryHeadline/>
+				<Summary>
```

Comment:
@Abban The `Summary` is basically a _switcher_. Do we still want to call it _Summary_? If we call it `Swithcer` then the possibility to use it for other use-cases opens up and the component becomes more reusable.

Reply:
Yes, summary is still a pattern, even though it leverages the switcher composition for it's layout. If you look at the pattern you can see there's some css required https://spenden.wikimedia.de/pattern-library/summary